app/workers/activitypub/low_priority_delivery_worker.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 class ActivityPub::LowPriorityDeliveryWorker < ActivityPub::DeliveryWorker
-  sidekiq_options queue: 'pull', retry: 8, dead: false
+  sidekiq_options queue: 'low_delivery', retry: 8, dead: false
```

Comment:
This may be over-analyzing, but I interpret `low_delivery` as "this is something which is unlikely to be delivered". Would `low_priority` or `nonessential` or `relaxed` or something (?) ...

Reply:
I originally named it for "Low priority delivery".
As I know, This sends these kind of activity
- Delete Actor
- Forward remotes' Delete activity

As it sends local "Delete Actor" activity, `nonessential` doesn't reflect about it.
In my opinion, I'd like `low_priority`

Reply:
Updated.